site/overview/package-overview.md

```diff
@@ -23,7 +23,7 @@ next:
 
 # Package Overview
 
-The Alchemy Account Kit SDK consists of a number of smaller packages that developers can leverage to interact with [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) infrastructure.
+Account Kit is comprised of a number of SDK packages that you can leverage to interact with [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) infrastructure.
```

Comment:
nit: prefer active voice over passive voice
```suggestion
Account Kit consists of a number of SDK packages that you can leverage to interact with [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) infrastructure.
```

Reply:
Yes! I could only find two pages for `aa-signers`
![image](https://github.com/alchemyplatform/aa-sdk/assets/83442423/8dd47d96-c799-469e-9abb-64d64545a3b4)

Reply:
today i merged in additional documentation. can you rebase and make sure those are up to style guidelines?

Reply:
actually just verified that they are there!